### ISSUE TITLE
Query about training setting.

### ISSUE THREAD

**Issue by Mollylulu** [OpenPerceptionX/PersFormer_3DLane#20]
Hi, thanks for your sharing.  I run the code for training, but do not get the results reported in your paper. Could you provide your training setting? Currently, I use your training script to train with `batch_size=2` & `gpu=4`.

**Comment by ilnehc** [comment]
You can find detailed training settings in the supplementary material. We train the model with `batch_size=8` & `gpu=8`.

**Comment by Mollylulu** [comment]
noted, I will go to check your supp material :)

**Comment by Mollylulu** [comment]
hi,  could you tell the learning rate scheduler of yours, which is not told in your material.  thanks !

**Comment by ilnehc** [comment]
The default setting is in the config file.
https://github.com/OpenPerceptionX/PersFormer_3DLane/blob/b863eaec4b2d1c6c42a9e99ebf3b8317ac3dd7d8/config/persformer_openlane.py#L81-L84

**Comment by Mollylulu** [comment]
noted, thanks !
